src/main/resources/db/migration/changes/Students.json

```diff
@@ -0,0 +1,74 @@
+{
+    "databaseChangeLog": [
+      {
+        "changeSet": {
+          "id": "1234",
+          "author": "Alec Song",
+          "preConditions": [
+            {
+              "onFail": "MARK_RAN"
+            },
+            {
+              "not": [
+                {
+                  "tableExists": {
+                    "tableName": "students"
+                  }
+                }
+              ]
+            }
+          ],
+          "changes": [
+            {
+              "createTable": {
+                "columns": [
+                  {
+                    "column": {
+                      "constraints": {
+                        "primaryKey": true,
+                        "primaryKeyName": "STUDENT_PK"
+                      },
+                      "autoIncrement": true,
+                      "name": "ID",
+                      "type": "BIGINT"
+                    }
+                  },
+                  {
+                    "column": {
+                      "name": "LNAME",
+                      "type": "VARCHAR(255)"
+                    }
+                  },
+                  {
+                    "column": {
+                      "name": "FMNAME",
```

Comment:
I think this would have to be FM_NAME if you keep the camel cased `fmName`.

I wonder if `firstMiddleName` and `FIRST_MIDDLE_NAME` would be better?